### PR TITLE
solution

### PR DESCRIPTION
**Pull request by BesConstantine**
- [DEMO LINK](https://besconstantine.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by roman-toritsyn — APPROVED** [review]
Годнота!

**Review by herkoss — CHANGES_REQUESTED** [review]
Well done!👏👍
But there are too large margins

Fix it please

Screenshot 2021-08-31 at 13 24 52

**Review by herkoss — CHANGES_REQUESTED** [review]
Well done!👏👍
But there are too large margins
Gap between cards should be 48px

Fix it please

Screenshot 2021-09-01 at 11 24 18

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
```
Comment (herkoss) [inline src/styles/_catalog.scss:18]: try other units

**Review by herkoss — CHANGES_REQUESTED** [review]

**Review by herkoss — APPROVED** [review]
Great!👍😊

**Pull request closed without merging**